src/styles/blocks/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  font-weight: 500;
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 50px;
+  background: $white-color;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0px 2px 4px 0px #0000000d;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), color values should be written in lowercase. Please, change `#0000000d` to `#0000000d`.